src/cryptography/hazmat/primitives/serialization/pkcs7.py

```diff
@@ -177,7 +178,88 @@ def sign(
         return rust_pkcs7.sign_and_serialize(self, encoding, options)
 
 
-def _smime_encode(
+class PKCS7EnvelopeBuilder:
+    def __init__(
+        self,
+        data: bytes | None = None,
+        recipients: list[x509.Certificate] | None = None,
```

Comment:
```suggestion
        *,
        _data: bytes | None = None,
        _recipients: list[x509.Certificate] | None = None,
```

People keep assuming these are public, let's make that harder for them.

Reply:
Fixed!